metricbeat/helper/prometheus/textparse.go

```diff
@@ -653,8 +658,6 @@ func ParseMetricFamilies(b []byte, contentType string, ts time.Time) ([]*MetricF
 			metricFamiliesByName[lookupMetricName] = fam
 		}
 
-		fam.Name = &metricName
```

Comment:
This line needed to be removed, because it was causing errors.

Example: if we had this metric:
```
# HELP cc_seconds A counter
# TYPE cc_seconds counter
# UNIT cc_seconds seconds
cc_seconds_total 1.0
cc_seconds_created 123.456
```

The metric would be renamed to `cc_seconds_total` and after to `cc_seconds_created`. When in reality, the metric name is `cc_seconds` (without the suffix). And then the `metric.Metric` is a list of metrics, where both are included.
This test case is in the test file.